apps/demo/emails/stack-overflow-tips.tsx

```diff
@@ -55,20 +55,20 @@ export default function Email({ tips = PropDefaults.tips }: EmailProps) {
             />
           </Section>
           <Section style={header}>
-            <Section style={headerContent}>
+            <Column style={headerContent}>
```

Comment:
Need to import `Column`

Reply:
Maybe do in a separate PR since it's not related to the Codepen

Reply:
Ok! I will separate in other PR, tkx

Reply:
Done